Test failures when running pytests locally with system timezone other than UTC

The following tests have issues with timezone differences when executing on a host system with the timezone other than UTC.

```
FAILED tests/test_acm/test_acm.py::test_certificate_expiration_status - AssertionError: assert 'EXPIRED' == 'ISSUED'
FAILED tests/test_glue/test_datacatalog.py::test_filter_crawls_for_start_time[LE] - assert 0 == 1
FAILED tests/test_glue/test_datacatalog.py::test_filter_crawls_for_start_time[LT] - assert 0 == 1
FAILED tests/test_glue/test_datacatalog.py::test_filter_crawls_for_start_time[GE] - AssertionError: assert 2 == 1
FAILED tests/test_glue/test_datacatalog.py::test_filter_crawls_for_start_time[GT] - AssertionError: assert 2 == 1
FAILED tests/test_glue/test_datacatalog.py::test_filter_crawls_for_end_time[LE] - assert 0 == 1
FAILED tests/test_glue/test_datacatalog.py::test_filter_crawls_for_end_time[LT] - assert 0 == 1
FAILED tests/test_glue/test_datacatalog.py::test_filter_crawls_for_end_time[GE] - AssertionError: assert 2 == 1
FAILED tests/test_glue/test_datacatalog.py::test_filter_crawls_for_end_time[GT] - AssertionError: assert 2 == 1
FAILED tests/test_glue/test_datacatalog.py::test_multiple_filters[LE] - assert 0 == 1
FAILED tests/test_glue/test_datacatalog.py::test_multiple_filters[LT] - assert 0 == 1
FAILED tests/test_glue/test_datacatalog.py::test_multiple_filters[GE] - AssertionError: assert 2 == 1
FAILED tests/test_glue/test_datacatalog.py::test_multiple_filters[GT] - AssertionError: assert 2 == 1
```

My development system is set for `America/Chicago` and produces these failures.

--- a/moto/acm/models.py
+++ b/moto/acm/models.py
@@ -1,4 +1,5 @@
 import base64
+import calendar
 import datetime
 import ipaddress
 import re
@@ -14,6 +15,7 @@ from cryptography.x509 import OID_COMMON_NAME, DNSName, IPAddress, NameOID
 from moto import settings
 from moto.core.base_backend import BackendDict, BaseBackend
 from moto.core.common_models import BaseModel
+from moto.core.serialize import parse_to_aware_datetime
 from moto.core.utils import utcnow
 
 from .exceptions import (
@@ -57,7 +59,8 @@ IPV4_REGEX = re.compile(
 
 
 def datetime_to_epoch(date: datetime.datetime) -> float:
-    return date.timestamp()
+    aware_dt = parse_to_aware_datetime(date)  # type: ignore
+    return float(calendar.timegm(aware_dt.timetuple()))
 
 
 class TagHolder(dict[str, Optional[str]]):

--- a/tests/test_glue/test_datacatalog.py
+++ b/tests/test_glue/test_datacatalog.py
@@ -8,6 +8,7 @@ from freezegun import freeze_time
 
 from moto import mock_aws, settings
 from moto.core import DEFAULT_ACCOUNT_ID as ACCOUNT_ID
+from moto.core.utils import utcnow
 
 from . import helpers
 
@@ -1576,7 +1577,7 @@ def test_filter_crawls_for_start_time(operator):
     helpers.create_crawler(client, name)
 
     client.start_crawler(Name=name)
-    between_the_two_run_starts = datetime.now()
+    between_the_two_run_starts = utcnow()
     # turn crawler to completed
     client.list_crawls(CrawlerName=name)
 
@@ -1608,7 +1609,7 @@ def test_filter_crawls_for_end_time(operator):
     client.list_crawls(CrawlerName=name)
 
     client.start_crawler(Name=name)
-    between_the_two_run_ends = datetime.now()
+    between_the_two_run_ends = utcnow()
     client.list_crawls(CrawlerName=name)
 
     crawls = client.list_crawls(
@@ -1636,7 +1637,7 @@ def test_multiple_filters(operator):
     # turn crawler to completed
     client.list_crawls(CrawlerName=name)
 
-    between_the_two_runs = datetime.now()
+    between_the_two_runs = utcnow()
     client.start_crawler(Name=name)
     client.list_crawls(CrawlerName=name)
 

